### ISSUE TITLE
Bottom cell not showing when scrolling

### ISSUE THREAD

**Issue by reejosamuel** [inamiy/YIFullScreenScroll#2]
1). Bottom cells gets cut off when scrolling to the last cell
2). The tabbar bounces the table hits the bottom with inertia.

I'm trying to fix both, if its easy for you kindly post a fix.

Thanks a ton

**Comment by inamiy** [comment]
Thanks for your feedback!
I think my commit `e35f74c` will solve (2), and hopefully (1) as well.

If you still find (1) issue, I also found similar bug in iOS6 beta a while ago, 
so please let me know and I will investigate further more.

**Comment by reejosamuel** [comment]
if you closely observe what pinterest or the likes are doing.

when the scroll view hits the bottom, they show the UI but they have set some content inset on the scrollview bottom

**Comment by inamiy** [comment]
My goal in this library is try not to modify the original scrollView's properties as best as possible
(though its contentInset will automatically change in iOS, 
 when using UITableViewController inside container-controllers & translucent UI-bars is set),
so I think it is "app's side" to properly handle its contentInset.

You can try 

```
self.tableView.contentInset = UIEdgeInsetsMake(0, 0, 44, 0);
```

at the end of `-viewDidLoad` in current demo app.
But please notice that additional padding will be made at bottom if you are using toolbar instead.

**Comment by inamiy** [comment]
I refactored code and updated version to 1.0.0.
This issue will be old one, but I guess it is already solved in 0.4.
Will close.
